README.md

```diff
@@ -33,7 +33,7 @@ This will allow you to make changes to your local repository of `leafygreen-ui`
 To do this, clone this repository and navigate to the root directory (where `package.json` is located), then run the following:
 
 ```
-npm run link -- ${PATH_TO_APPLICATION}
+yarn run link -- ${PATH_TO_APPLICATION}
```

Comment:
Can we test this please?

Reply:
Tested this and it works! A `package-lock.json` is created in the process though, and trying to figure out where that's happening and how to prevent.

Reply:
Update: fixed this!